docs/pages/guides/using-nextjs.mdx

```diff
@@ -143,242 +242,43 @@ This is Vercel's preferred method for deploying Next.js projects to production.
-}
-```
-
-</Collapsible>
-
-[next-images]: https://github.com/twopluszero/next-images
-[next-optimized-images]: https://github.com/cyrilwanner/next-optimized-images
-
-### Font support
-
-By default Next.js doesn't support static assets like an Expo project. Because this is the intended functionality of Next.js, `@expo/next-adapter` doesn't add font support by default. If you want to use libraries like `expo-font`, `@expo/vector-icons`, or `react-native-vector-icons` you'll need to change a few things.
-
-- Install the plugin - `yarn add next-fonts`
-  - [`next-fonts`][next-fonts] injects a webpack loader to handle fonts.
-- Wrap the font method with the Expo method in your **next.config.js**:
-
-  - The order is important because Expo can mix in the location of vector icons to the existing font loader.
-
-  ```js
-  const { withExpo } = require('@expo/next-adapter');
-  const withFonts = require('next-fonts');
-
-  module.exports = withExpo(
-    withFonts({
-      projectRoot: __dirname,
-    })
-  );
-  ```
-
-- Now restart your project and you should be able to load fonts!
-
-You can test your config with the following example:
-
-<Collapsible summary="Show Example">
-
-```js
-import React, { useEffect, useState } from 'react';
-import * as Font from 'expo-font';
-import { Text } from 'react-native';
-
-export default function FontDemo() {
-  const [loaded, setLoaded] = useState(false);
-
-  useEffect(() => {
-    (async () => {
-      try {
-        await Font.loadAsync({
-          // You can get this font on GitHub: https://shorturl.at/chEHS
-          'space-mono': require('./assets/SpaceMono-Regular.ttf'),
-        });
-      } catch ({ message }) {
-        // This will be called if something is broken
-        console.log(`Error loading font: ${message}`);
-      } finally {
-        setLoaded(true);
-      }
-    })();
-  }, []);
-
-  if (!loaded) return <Text>Loading fonts...</Text>;
-
-  return <Text style={{ fontFamily: 'space-mono' }}>Hello from Space Mono</Text>;
-}
-```
-
-</Collapsible>
-
-[next-fonts]: https://github.com/rohanray/next-fonts
-
-## API
-
-### CLI
-
-Generate static Next.js files into your project.
-
-#### CLI Options
-
-For more information run `yarn next-expo --help` (or `-h`)
+## Limitations or differences comparing to the default Expo for Web
 
-| Shortcut | Flag          | Description                                           |
-| -------- | ------------- | ----------------------------------------------------- |
-| `-f`     | `--force`     | Allows replacing existing files                       |
-| `-c`     | `--customize` | Select template files you want to add to your project |
-| `-V`     | `--version`   | output the version number                             |
+Using Next.js for the web means you will be bundling with the Next.js Webpack config. This will lead to some core differences in how you develop your app vs your website.
```

Comment:
```suggestion
Using Next.js for the web means you will be bundling with the Next.js webpack config. This will lead to some core differences in how you develop your app vs your website.
```

Fixed the lint case police for "Webpack" -> "webpack".